backend/forgather/docs/자동_토큰_갱신_README.md

```
@@ -0,0 +1,370 @@
+- 토큰 갱신이 백그라운드에서 자동 처리
+- 더 깔끔하고 유지보수하기 쉬운 코드
+
+## 테스트 가이드
+
+### 1. 기본 토큰 갱신 테스트
+
+1. 어드민 패널에 로그인
+2. 브라우저 DevTools → Application → Local Storage 열기
+3. Access Token을 수동으로 만료시키기:
+   - Access Token 값 복사
+   - https://jwt.io 에서 디코딩
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)